src/ethereum/rfc003/erc20_htlc.rs

```diff
@@ -78,11 +79,11 @@ mod tests {
     fn compiled_contract_is_same_length_as_template() {
         let htlc = Erc20Htlc::new(
             3_000_000,
-            Address::default(),
-            Address::default(),
+            Address([0u8; 20]),
+            Address([0u8; 20]),
```

Comment:
Perhaps a constructor method `zero`

Reply:
Yeah we thought about it but we decided to keep it simple as these are just tests.
In production you would never build a `default` or `zero` address but convert your address to this struct.